cmd/notification.go

```diff
@@ -253,7 +254,7 @@ func (sys *NotificationSys) Init(objAPI ObjectLayer) error {
 	}
 
 	for _, bucket := range buckets {
-		config, err := readNotificationConfig(objAPI, bucket.Name)
+		config, err := readNotificationConfig(context.Background(), objAPI, bucket.Name)
```

Comment:
The correct approach is `Init()` would need to accept `context` and use it.  This way caller has more control on initialization process.

Reply:
@balamurugana, context has been defined for the new ObjectLayer Api calls as placeholders. I do know that it would be passed all the way from Init(). However it is outside the scope of this PR since context addition/error logging is being added as a separate PR. Init() method will itself need to inherit context when object layer is initialized, which will cascade into a larger change than the specific focus of this PR.

Reply:
As you change/introduce context in `Init()`, I do not understand why passing context to `Init()` is out of scope of this PR.

I will check again once you address all other comments,